lib/sqs/index.js

```diff
@@ -36,13 +36,30 @@ module.exports = ({
   defaultVisibilityTimeout = 300,
   bypassProxy = true,
   longPollingWaitTime = 20,
-  sqsEndpoint = null
+  sqsEndpoint = null,
+  sqsTimeout = null,
+  sqsConnectTimeout = null
 }) => {
   let ep = {};
   if (sqsEndpoint) {
     ep = { endpoint: new AWS.Endpoint(sqsEndpoint) };
   }
-  const SQS = setHttpAgent({ ...clientConfig, ...ep }, bypassProxy, AWS.SQS);
+  const extraHttpOptions = {};
+  if (sqsTimeout) {
+    extraHttpOptions.timeout = sqsTimeout;
+  }
+  if (sqsConnectTimeout) {
+    extraHttpOptions.connectTimeout = sqsConnectTimeout;
+  }
+  const SQS = setHttpAgent(
+    {
+      ...clientConfig,
+      ...ep,
+      httpOptions: { ...clientConfig.httpOptions, ...extraHttpOptions }
```

Comment:
what happens when client config has http options and we also pass sqsTimeout and sqsConnectTimeout simultaneously?

Reply:
We will merge the client config's http options with the extra http options